config/helpers.sh

```diff
@@ -1,10 +1,19 @@
 # shellcheck shell=bash
 # Other settings and helper functions.
 
-debug_mode_enabled() {
+# shellcheck source=lib/util.sh
+source "${TMUX_POWERLINE_DIR_LIB}/util.sh"
+
+tp_debug_mode_enabled() {
 	[ -n "$TMUX_POWERLINE_DEBUG_MODE_ENABLED" ] && [ "$TMUX_POWERLINE_DEBUG_MODE_ENABLED" != "false" ]
```

Comment:
Ah good thinking. Could you drop a note in [CHANGELOG.md](https://github.com/erikw/tmux-powerline/blob/main/CHANGELOG.md) under "Unreleased" and add a sub-header "Changed" as in some other places below, something like
```markdown
- Deprecation Warning: function `patched_font_in_use()` will be replaced by `patched_font_in_use()` in future releases. Please update your custom themes and segments now. [#489](https://github.com/erikw/tmux-powerline/pull/489)
```